composer.json

```diff
@@ -32,10 +32,12 @@
     "scripts": {
         "compat": "./vendor/bin/phpcs --standard=phpcs-compat-ruleset.xml .",
         "phpcs": "./vendor/bin/phpcs --standard=phpcs-ruleset.xml -s .",
+        "phpcs-tests": "./vendor/bin/phpcs --standard=phpcs-test-ruleset.xml -s ./tests/",
```

Comment:
Add a specific code standard test config to Circle. Allows for a higher version of PHP.